Gruntfile.js

```diff
@@ -25,6 +26,8 @@ module.exports = function (grunt) {
       }
     },
 
+    clean: ['tmp', 'ember-dist', 'node_modules', 'bower_components'],
```

Comment:
I guess this PR is mostly a convenience for me. I find myself `rm`'ing these folders a lot, so I can reinstall and rerun from scratch. I mostly use it when we update dependencies.